data/hai/hai reveal 4 middle.txt

```diff
@@ -26,7 +26,7 @@ mission "Hai Reveal [A00] Darkwaste Timer"
 	destination Darkwaste
 	to offer
 		"hai slave prereq" >= 3
-		"combat rating" > 3000
+		"combat rating" > 2400
```

Comment:
We'll need to mention in an earlier mission that the player needs more combat experience, if they don't have 2400.  (@Hurleveur pointed this out to me a few weeks ago.) Otherwise, they'll wander into the battle, an be unable to finish it. Then they'll have to go out and kill a lot of people to gain experience while keeping their escorts alive.

Reply:
Do any of the escorts wander out to fight them? That'll separate them from the main fleet, letting them be swarmed and killed.

Reply:
Tribute fleets are always ignored by all govs except the player

Reply:
I was referring to the random encounter fleets. I put that comment in the wrong comment chain.